CHANGELOG.rst

```diff
@@ -2,10 +2,12 @@ Changelog
 =========
 
 
-1.4.0 (unreleased)
+2.0.0 (unreleased)
 ------------------
 
-- Nothing changed yet.
+**Breaking changes**
+
+- Upgrade to Kinto 13 api. Redis keys changed. ``*.records`` became ``*.objects``.
```

Comment:
`API` upercase.

What does it mean in terms of migrations if Redis keys were changed?

Reply:
I was going to assume people we start with an empty kinto and then would rename the keys, but I guess we can either keep the previous keys (and not rename it) or create a script to rename them. What do you think?

Reply:
I don't even know if we have users of kinto-redis :) So least effort seems good to me :)

Reply:
I am gonna keep the previous keys. I guess except me we probably don't because nobody noticed it wasn't working with Kinto 13 :dancer: